packages/components/src/Components/TagCount/tagCount.scss

```diff
@@ -0,0 +1,13 @@
+.ins-c-tagCount {
+    color: #9a9a9a;
```

Comment:
I can't find this color in pf color palet [patternfly-4/variables](https://patternfly-react.surge.sh/patternfly-4/tokens/Global%20CSS%20variables/) I guess it would be better to use `var(--pf-global--icon--Color--light)` instead of custom color.